### PR TITLE
Bugfix/fix color theme import cycle

### PR DESCRIPTION
**Pull request by gazizova-vlada**
- Moving color theme components from the Common module to the Components module
- Move style components to color theme components so that inside the Components module there is no import from the Common module

### CODE REVIEW

File: packages/client/src/components/panels/UploadPanel/SubComponents/LoadingButton.js [hunk 1]
```diff
@@ -1,11 +1,83 @@
...
+const rotate360 = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+const StyledLoadingButton = styled.div`
```
Comment (AlexeySafronov) [inline packages/client/src/components/panels/UploadPanel/SubComponents/LoadingButton.js:16]: Moved styled code to separated files

File: packages/components/ColorTheme/styled/filterBlockItemTag.js [hunk 2]
```diff
@@ -1,7 +1,32 @@
 import styled, { css } from "styled-components";
-import { StyledFilterBlockItemTag } from "@docspace/common/components/FilterInput/sub-components/StyledFilterBlock";
 import Base from "@docspace/components/themes/base";
 
+const StyledFilterBlockItemTag = styled.div`
```
Comment (AlexeySafronov) [inline packages/components/ColorTheme/styled/filterBlockItemTag.js:4]: Move styled code to separated file

File: packages/components/ColorTheme/styled/floatingButton.js [hunk 3]
```diff
@@ -1,35 +1,39 @@
 import styled, { css } from "styled-components";
-import {
-  StyledCircleWrap,
-  StyledFloatingButton,
-  IconBox,
-  StyledCircle,
-} from "@docspace/common/components/FloatingButton/StyledFloatingButton";
 import Base from "@docspace/components/themes/base";
 
+const StyledCircleWrap = styled.div`
```
Comment (AlexeySafronov) [inline packages/components/ColorTheme/styled/floatingButton.js:4]: Move styled code to separated file

File: packages/client/src/components/panels/UploadPanel/SubComponents/LoadingButton.js (hunk 1 above)
Comment (gazizova-vlada) [reply packages/client/src/components/panels/UploadPanel/SubComponents/LoadingButton.js]: Moved styles to one file StyledLoadingButton

**Pull request merged**